### PR TITLE
Added support for Govee H5100

### PR DESCRIPTION
**Pull request by realzhiqiang**
## Description:

Govee H5100 use same format as those listed in H5101/02/74/77.  Updated H5102_json.h to support Govee H5100

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] I accept the [DCO](https://github.com/theengs/decoder/blob/development/docs/participate/development.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by DigiH** [comment]
Thanks @realzhiqiang!

Could you also add a documentation page for the H5100, similar to the ones for H5101.md, H5102.md etc. so that it will show up in the [compatibility listing](https://decoder.theengs.io/devices/devices.html)?

**Comment by realzhiqiang** [comment]
> Thanks @realzhiqiang!
> 
> Could you also add a documentation page for the H5100, similar to the ones for H5101.md, H5102.md etc. so that it will show up in the [compatibility listing](https://decoder.theengs.io/devices/devices.html)?

added. Thanks

File: docs/devices/H5100.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Govee H5100
+
+|Model Id|[H5100](https://github.com/theengs/decoder/blob/development/src/devices/H5102_json.h)|
+|-|-|
+|Brand|Govee|
+|Model|Smart Thermo Hygrometer|
+|Short Description|Indoor Digital Humidity and Temperature monitor with LCD display|
```
Comment (DigiH) [inline docs/devices/H5100.md]: @realzhiqiang

Does it also have a display? I just had a quick look and think it is without display, so probably just
"Indoor Digital Humidity and Temperature monitor"

File: docs/devices/H5100.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+|Communication|BLE broadcast|
+|Frequency|2.4Ghz|
+|Power source|2 AAA|
```
Comment (DigiH) [inline docs/devices/H5100.md]: @realzhiqiang

I think it only has 1 AAA? Can you confirm this?

File: docs/devices/H5100.md (hunk 1 above)
Comment (DigiH) [reply docs/devices/H5100.md]: Done

File: docs/devices/H5100.md (hunk 2 above)
Comment (DigiH) [reply docs/devices/H5100.md]: Done

**Review by DigiH — APPROVED** [review]

File: docs/devices/H5100.md (hunk 2 above)
Comment (realzhiqiang) [reply docs/devices/H5100.md]: Hi, sorry should have pay more attention about the differences between other variants. 

Yes, it does not have a display, use sonly 1 AAA battery.

Comment (DigiH) [reply docs/devices/H5100.md]: I have made the changes and merged the PR.

Thanks again for this addition :)

**Pull request merged**